app.js

```diff
@@ -58,7 +52,68 @@ var itemObjects = [
 ];
 
 var itemsObjectsWorking;
-var itemsChosen = 0;
+var itemsChosen = window.localStorage.itemsChosen || 0;
+var itemsDisplayed = window.localStorage.itemsDisplayed || 0;
```

Comment:
These values will come out of `localStorage` as `string`s. You could use `parseInt`, but the absolute simplest way to force any value to an integer (or 0, if invalid) is `|0`, i.e. `window.localStorage.itemsChosen|0`.

You may find [asm.js](http://asmjs.org/) of interest, and it uses `|0` as its [`int` type annotation](http://asmjs.org/spec/latest/#parameter-type-annotations).